### PR TITLE
Incorporate translations (received from BOSA) for the new strings in OAT

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-126

**Description**

As BOSA author
I would like the new functionality of Tabular Import to be provided with translations for French & Dutch
so that I can use it on my native language. 

**Acceptance Criteria** 

- AC1

```
GIVEN provided French translation for new keys introduced during Tabular Import feature implementation
   AND author is logged-in with preferred interface language set to French
THEN  TAO shall present Tabular Import related strings in French according to the translations provided
```

- AC2

```
GIVEN provided Dutch translation for new keys introduced during Tabular Import feature implementation
   AND author is logged-in with preferred interface language set to Dutch
THEN  TAO shall present Tabular Import related strings in Dutch according to the translations provided
```

**Environment**

http://aadvanced-search-integration.playground.kitchen.it.taocloud.org:48321

### CODE REVIEW

File: model/import/Report/WarningReportFormatter.php [hunk 1]
```diff
@@ -34,10 +34,7 @@ class WarningReportFormatter extends AbstractReportFormatter
     public function format(array $report): Report
     {
         $reportObject = Report::createWarning(
-            __(
-                '%s line(s) are imported with warnings',
-                count($report)
-            )
+            __('%s line(s) are imported with warnings', count($report))
```
Comment (gabrielfs7) [inline model/import/Report/WarningReportFormatter.php:37]: Can we revert this change? I do not see the reason to change it.

File: model/import/Report/ErrorReportFormatter.php [hunk 2]
```diff
@@ -34,10 +34,7 @@ class ErrorReportFormatter extends AbstractReportFormatter
     public function format(array $report): Report
     {
         $reportObject = Report::createError(
-            __(
-                '%s line(s) contain(s) an error and cannot be imported',
-                count($report)
-            )
+            __('%s line(s) contain(s) an error and cannot be imported', count($report))
```
Comment (gabrielfs7) [inline model/import/Report/ErrorReportFormatter.php:37]: Not needed change, lets revert.

https://github.com/oat-sa/extension-tao-itemqti/pull/1755/files#r643008991

File: model/import/CsvItemImporter.php [hunk 3]
```diff
@@ -90,10 +90,7 @@ public function import($class, $form, $userId = null)
             $report = $reportBuilder->buildReportsContainer(__('CSV import failed'), $errorMessage);
             $this->getLogger()->warning($errorMessage);
         } catch (Throwable $e) {
-            $errorMessage = __(
-                'An unexpected error occurred during the CSV import. The system returned the following error: "%s"',
-                $e->getMessage()
-            );
+            $errorMessage = __('An unexpected error occurred during the CSV import. The system returned the following error: "%s"', $e->getMessage());
```
Comment (gabrielfs7) [inline model/import/CsvItemImporter.php:93]: Not needed change, lets revert it, please

**Review by gabrielfs7 — APPROVED** [review]
LGTM, but please revert the PHP changes, cause they are not required and we should do like that for big lines.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: model/import/Report/WarningReportFormatter.php (hunk 1 above)
Comment (habemuscode) [reply model/import/Report/WarningReportFormatter.php:37]: The reason of the change is that translation script doesn't work with previous markup. We need to fix translator in a separated tech task

cc: @boajer

Comment (gabrielfs7) [reply model/import/Report/WarningReportFormatter.php:37]: Thanks for the clarification!

**Review by boajer — APPROVED** [review]
Indeed we have an [issue with a translation parser](https://github.com/oat-sa/tao-core/blob/ea9abe98edbddc7d75df0c212dfb68c3b00b32e6/helpers/translation/class.SourceCodeExtractor.php#L226) that due to urgency of this PR may need to be fixed separately

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by gabrielfs7** [comment]
I am closing this PR, cause it was already incorporated in another PR https://github.com/oat-sa/extension-tao-itemqti/pull/1773
